docs/developers/omnichain/zrc-20.md

```diff
@@ -16,6 +18,17 @@ place extremely simple — as if they were all on a single chain.
 
 ![zeta-evm](/img/graphs/zrc-20-header.svg)
 
+## Supported Assets
+
+I list of currently supported assets is stored on-chain:
+
+<ForeignCoinsTable />
+
+New assets can be added or removed by broadcasting a transaction with a
+corresponding message of the
+[`fungible` module](http://localhost:3001/architecture/modules/fungible/messages/)
```

Comment:
localhost??

Reply:
https://github.com/zeta-chain/docs/pull/197/commits/01128af14ecafd180fcde94111e837e14130dcc6

Reply:
@andresaiello should be fixed now.